src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -67,6 +67,8 @@ public class ConsoleOptions : CommandLineOptions
 
         public bool LoadUserProfile { get; private set; }
 
+        public string ConsoleEncoding { get; private set; }
```

Comment:
This probably belongs in CommandLineOptions, I expect that if we accept this PR, we will want to do the same for NUnitLite and the .NET Core runner.